db/migrate/20161127084744_remove_featuredfrom_skins.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveFeaturedfromSkins < ActiveRecord::Migration
+  def up
+    remove_column :skins, :featured
+  end
+
+  def down
+    add_column :skins:, :featured
```

Comment:
Hound is right -- there's a `:` that doesn't belong here. Other than that, looks good!